python/cuml/cuml/tests/test_fil.py

```diff
@@ -12,13 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# flake8: noqa
```

Comment:
Why is this needed?

Reply:
This is because I moved the XGBoost import earlier, as part of the workaround for the wheel test

Reply:
Ok, can we limit it to the specific error code?

Reply:
It's currently not possible to exclude specific warnings for an entire file. https://stackoverflow.com/questions/48153886/flake8-ignore-specific-warning-for-entire-file

Reply:
I see, that's annoying. Let's just add a short comment above or below that explains why this was added.

Reply:
Alternatively, we add it here with a comment:

https://github.com/rapidsai/cuml/blob/1e0297dbe72fb4ca6fb5d86004e94780f0e520c9/python/cuml/.flake8#L30-L40

Reply:
I think it should be possible to `#noqa` just the line where the import is being done?

Reply:
I was able to disable specific checks by adding `# noqa: E402` to each line of import.